tensorboard/loader.py

```diff
@@ -399,6 +401,254 @@ def __str__(self):
     return u'BufferedRecordReader{%s}' % self.path
 
 
+class RateCounter(object):
+  """Utility class for tracking how much a number increases each second.
```

Comment:
This class seems to assume throughout that the value that it's tracking is an integer (the tail-call to `int(…)` in `get_rate`, the `%d` format specifiers in error messages). If this is what you want, it's probably worth documenting here: just `s/a number/an integer`. Otherwise, it'd be easy to change.